build/build.js

```diff
@@ -103,3 +102,13 @@ processFile(
   , path.join(testourroot, '../common.js')
   , testReplace['common.js']
 )
+
+//--------------------------------------------------------------------
+// Update Node version in README
+
+processFile(readmePath, readmePath, [
+  [
+      RegExp('(Node-core v)' + nodeVersionRegexString, 'g')
```

Comment:
you can reuse the same regex you used to test above and have it just [versionRegex, nodeVersion]